roles/etcd/handlers/main.yml

```diff
@@ -21,6 +21,8 @@
 - name: wait for etcd up
   uri:
     url: "https://{% if is_etcd_master %}{{ etcd_address }}{% else %}127.0.0.1{% endif %}:2379/health"
+    client_cert: "{{ etcd_cert_dir}}/admin-{{ groups['etcd'][0] }}.pem"
+    client_key: "{{ etcd_cert_dir }}/admin-{{ groups['etcd'][0] }}-key.pem"
     validate_certs: no
```

Comment:
should we put that to true ?

Reply:
It does not validate :( and there is no way to specify an explicit cert